perf.groovy

```diff
@@ -639,13 +651,11 @@ parallel(
 
                     if (isPR) {
                         def opts = ""
-                        if (opt_level == 'min_opts')
-                        {
+                        if (opt_level == 'min_opts') {
```

Comment:
I haven't found any, but this file have them all mixed up. Assuming Java style, I reformatted the doc for consistency.